README.md

```diff
@@ -21,9 +21,8 @@ AWS_ACCESS_KEY_ID=foo AWS_SECRET_ACCESS_KEY=foo sbt test
 ```
 
 ## Publishing snapshots
-Read this [article](http://www.loftinspace.com.au/blog/publishing-scala-libraries-to-sonatype.html#publish_snapshot) for how to configure signing.
-Once that is done, all one needs to run is:
-
+Snapshots will now be published to our local maven repo in s3.
 ```
-sbt publishSigned
+sbt publish
```

Comment:
Same as `spark-hyperloglog`, let's leave this out to prevent overwriting the auto-published jars.